gitreload/config.py

```diff
@@ -1,38 +1,45 @@
 """
 Setup configuration from a json file with defaults
 """
-import json
 import logging
 from logging.handlers import SysLogHandler
 import os
 import platform
 
 log = logging.getLogger('gitreload')  # pylint: disable=C0103
 
-CONFIG_PATHS = [
-    os.environ.get('GITRELOAD_CONFIG', ''),
-    os.path.join(os.getcwd(), 'gr.env.json'),
-    os.path.join(os.path.expanduser('~'), '.gr.env.json'),
-    '/etc/gr.env.json',
-]
+MINUTE = 60  # seconds
 
 
-def configure_logging(level_override=None):
+class Config(object):
```

Comment:
General question: have you removed the ability to configure the app via a JSON file, and now it's only possible to configure via OS environment variables? I personally find JSON config a _lot_ easier to manage. Seems like we could have both if we attempted to load a JSON file from some path(s) before defining this class, then adding a helper method that tries to get the value from the OS environment, then the JSON config file dict if that doesn't exist, then a default value if that doesn't exist.

Reply:
I talked about it with Tobias and he agreed on the change

Reply:
OK